interface-definitions/protocols_openfabric.xml.in

```diff
@@ -0,0 +1,188 @@
+        <properties>
+          <help>OpenFabric protocol</help>
+          <priority>680</priority>
+        </properties>
+        <children>
+          <tagNode name="interface">
+            <properties>
+              <help>Interface params</help>
+              <completionHelp>
+                <script>${vyos_completion_dir}/list_interfaces</script>
+              </completionHelp>
+              <valueHelp>
+                <format>txt</format>
+                <description>Interface name</description>
+              </valueHelp>
+              <constraint>
+                #include <include/constraint/interface-name.xml.i>
+              </constraint>
+            </properties>
+            <children>
+              <leafNode name="csnp-interval">
+                <properties>
+                  <help>Set Complete Sequence Number Packets (CSNP) interval</help>
+                  <valueHelp>
+                    <format>u32:1-600</format>
+                    <description>CSNP interval in seconds</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 1-600"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+              <leafNode name="hello-interval">
+                <properties>
+                  <help>Set Hello interval</help>
+                  <valueHelp>
+                    <format>u32:1-600</format>
+                    <description>Hello interval in seconds</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 1-600"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+              <leafNode name="hello-multiplier">
+                <properties>
+                  <help>Set multiplier for Hello holding time</help>
+                  <valueHelp>
+                    <format>u32:2-100</format>
+                    <description>Multiplier for Hello holding time</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 2-100"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+              <leafNode name="metric">
+                <properties>
+                  <help>Set interface metric value</help>
+                  <valueHelp>
+                    <format>u32:0-16777215</format>
+                    <description>Interface metric value</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 0-16777215"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+              <leafNode name="passive">
+                <properties>
+                  <help>Configure passive mode for interface</help>
```

Comment:
This help string doesn't explain anything, to be fair. "Do not initiate adjacencies"?
There certainly shouldn't be the word "Configure" in these help strings.